android/src/fdroid/play/release-notes/en-US/default.txt

```diff
@@ -1,9 +1 @@
-• Updated OSM maps 🗺️ data to 210825
-• Max number of intermediate stops is increased from 3 to... try to guess :)
-• Finally, maps download should work for everyone! And if not, please tell us.
-• Bookmark button now shows "Save" and "Delete" instead of just "Bookmark"
-• Farmlands are visible now on the map!
-• 🇧🇷 Brazil and 🇵🇹 Portuguese translations are updated
-• Fixed bug with speed camera icon
-• Fixed some non-displayed texts on the map
-• Fixed crashes with some deleted map types/objects
+• Fixed missing buttons on toolbar after the search
```

Comment:
Fixed WiFi toggling when editing phones

Reply:
I spent 10 minutes trying to understand what was fixed. This was a really minor issue.